### PR TITLE
example sentences for additional senses

### PR DESCRIPTION
**Pull request by jacob-8**
#### Relevant Issue
(prepend "closes" if issue will be closed by PR)

#### How can the changes be tested? 
Please also provide applicable links using relative paths from root (e.g. `/apatani/entries`) and reviewers can just add that onto preview urls or localhost.

#### Checklist before marking ready to merge
Please keep it in draft mode until these are completed:
- [ ] Equal time was spent cleaning the code as writing it (Boy Scout Rule)
  - [ ] Functions
    - [ ] Functions that don't belong in Svelte components are extracted out into `.ts` files
    - [ ] Functions are short and well named
    - [ ] Concise tests are written for all functions
  - [ ] Classes (a Svelte Component is a Class)
    - [ ] Svelte components are broken down into smaller components so that each component is responsible for one thing (Single Responsibility Principle)
    - [ ] Stories/variants are written to describe use cases
  - [ ] Comments are only included when absolutely necessary information that cannot be explained in code is needed

### CODE REVIEW

File: .vscode/svelte.code-snippets [hunk 1]
```diff
@@ -0,0 +1,262 @@
+{
+	// Place your living-dictionaries workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and
```
Comment (Danble) [inline .vscode/svelte.code-snippets:2]: @jacob-8 what is this for?

File: supabase/migrations/20240223012557_sentences.sql [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  updated_by uuid NOT NULL REFERENCES auth.users (id),
+  deleted timestamp with time zone
+);
+
+ALTER TABLE sentences ENABLE ROW LEVEL SECURITY;
+
+CREATE TABLE senses_in_sentences (
+  sense_id uuid NOT NULL REFERENCES senses (id),
+  sentence_id uuid NOT NULL REFERENCES sentences (id),
+  created_by uuid NOT NULL REFERENCES auth.users (id),
```
Comment (Danble) [inline supabase/migrations/20240223012557_sentences.sql:17]: I think I understand the reason behind this table: having a many to many relation between senses and sentences.
But not sure about the reason why this table has the fields created_by and updated_at too.
Is it due to the relationship can be updated? like changing a sentence from sense 2 to another sense like sense 4 for example? @jacob-8

File: supabase/migrations/20240223012557_sentences.sql [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  new_value text, 
+  old_value text
+);
+
+ALTER TABLE sentence_updates ENABLE ROW LEVEL SECURITY;
+
+CREATE OR REPLACE FUNCTION ensure_dictionary_before_apply_sentence_updates()
+RETURNS TRIGGER AS $$
+DECLARE
+  column_data_type regtype;
```
Comment (Danble) [inline supabase/migrations/20240223012557_sentences.sql:48]: @jacob-8 Why are you declaring these variables if you're not using them here? 
Are they for future changes?

File: supabase/migrations/20240223012557_sentences.sql (hunk 2 above)
Comment (jacob-8) [reply supabase/migrations/20240223012557_sentences.sql:17]: Yes relationships can be added and removed from any sense to any sense. We want to know who made the change and when.

File: .vscode/svelte.code-snippets (hunk 1 above)
Comment (jacob-8) [reply .vscode/svelte.code-snippets:2]: Gives you snippets that used to be provided by an extension. The extension grew stale and was missing important things like const and key so I just added the snippets to the repo and now the extension is no longer needed. Plus I have my favorite Vitest snippets in there too. Please try them out and use them.

File: supabase/migrations/20240223012557_sentences.sql (hunk 2 above)
Comment (Danble) [reply supabase/migrations/20240223012557_sentences.sql:17]: Ok, Thanks!

**Comment by Danble** [comment]
@jacob-8 What's the reason for moving Open Graph from parts to site?

**Pull request merged**